test/closest.test.js

```diff
@@ -0,0 +1,52 @@
+import domassist from '../domassist';
+import test from 'tape-rollup';
+
+function addNode(el, num) {
```

Comment:
Although this test is similar than the previous one, I think we should just have a template with a clear structure on how it looks and what's the expected result. Look at what we do with Domodules. What do you think about it?